mongodb.cpp

```diff
@@ -185,6 +185,7 @@ static class MongoDBExtension : public Extension {
 			/* MongoDb\Driver\ReadConcern */
 			HHVM_MALIAS(MongoDB\\Driver\\ReadConcern, __construct, MongoDBDriverReadConcern, __construct);
 			HHVM_MALIAS(MongoDB\\Driver\\ReadConcern, __debugInfo, MongoDBDriverReadConcern, __debugInfo);
+			HHVM_MALIAS(MongoDB\\Driver\\ReadConcern, bsonSerialize, MongoDBDriverReadConcern, __debugInfo);
```

Comment:
It was certainly on purpose, and I suspect jsonSerialize() would be exactly the same indeed.